src/utils/file_utils.py

```diff
@@ -16,10 +21,19 @@ def read_yaml_file(file_path: str) -> Dict:
     return data
 
 
-def get_files_from_folder_path(path: str) -> List:
-    return [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+def get_files_from_folder_path(path: Path) -> Result[List, str]:
+    if check_if_path_exists(path):
+        files = [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+        logger.info(f"Found {len(files)} in {path}")
+        logger.debug(f"Files are: {files}")
+        return Ok(files)
+
+    return Err(f"{path} does not exist.")
+
 
+def get_file_name_from_path(file_path: Path) -> str:
```

Comment:
I think the usages of this function might've been removed. Also, does it provide any value?

Reply:
It is still being used, but it is true that it does not provide much value... Will remove it.